packages/node_modules/@webex/plugin-messages/test/integration/spec/messages.js

```diff
@@ -207,6 +207,62 @@ describe('plugin-messages', function () {
               validateMessageEvent(event, message, actor);
             }));
       });
+
+      it('posts a message to a card to a room validates the event', () => {
+        const created = new Promise((resolve) => {
+          webex.messages.on('created', (event) => {
+            debug('message created event called');
+            resolve(event);
+          });
+        });
+        const attachment = {
+          contentType: 'application/vnd.microsoft.card.adaptive',
+          content: {
+            type: 'AdaptiveCard',
+            version: '1.0',
+            body: [
+              {
+                type: 'TextBlock',
+                text: 'Here is an image'
+              },
+              {
+                type: 'Image',
+                url: KNOWN_HOSTED_IMAGE_URL,
+                size: 'small'
+              }
+            ]
+          }
+        };
+
+        return webex.messages.listen()
+          .then(() => webex.messages.create({
+            roomId: room.id,
+            text,
+            attachments: [attachment]
+          })
+            .then(async (message) => {
+              validateMessage(message, text, 0, attachment);
```

Comment:
Looks like all the assertions are nested here, could you add a comment above this line:
```js
// Assert that the message shape is valid and contains attachment data.
```
This will make finding the section in which assertions are made a bit clearer for future implementations that utilize this method.